src/app/feed/feed-create-post/feed-create-post.component.html

```diff
@@ -41,6 +41,29 @@
         <i class="icon-close feed-post__image-delete" (click)="postImageSrc = null"></i>
         <img class="feed-post__image" src="{{ postImageSrc }}" />
       </div>
+      <!-- Video is processing - please wait -->
+      <div class="d-flex flex-column align-items-center" *ngIf="!isComment && postVideoSrc && !readyToStream">
+        <div>Video Processing In Progress</div>
+        <simple-center-loader [height]="150"></simple-center-loader>
+      </div>
+      <!-- Video Player -->
+      <div
+        *ngIf="!isComment && postVideoSrc && readyToStream"
+        class="feed-post__video-container"
+      >
+        <iframe
+          [src]="postVideoSrc | sanitizeVideoUrl"
+          allow="accelerometer; gyroscope; autoplay; encrypted-media; picture-in-picture;"
+          allowfullscreen
+          class="feed-post__video"
+        >
+        </iframe>
+      </div>
+      <!--        <cloudflare-stream-->
```

Comment:
Do we need this?

Reply:
no - I was testing out whether I preferred the cloudflare-stream component or an iframe. I think the iframe is better long term as videos are uploaded to other locations. Don't want to dirty up the codebase trying to figure out if we should use this component or an iframe. We may need to write a utility that tests out whether or not an iframe is going to work for the URL provided.